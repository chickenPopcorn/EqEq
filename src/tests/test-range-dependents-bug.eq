Pendulum = {
  magiceq = eqparam;
}

Pendulum:find magiceq with eqparam in range(5); {
  print("magiceq: %.0f\n", magiceq);
}
