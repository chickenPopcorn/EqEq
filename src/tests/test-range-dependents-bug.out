magiceq: 0
magiceq: 1
magiceq: 2
magiceq: 3
magiceq: 4
magiceq: 5
